### ISSUE TITLE
Direct Message-

### ISSUE THREAD

**Issue by Morocco-germany** [andrewsegas/instaboost#16]
> Fill in all requirements *before* submitting an issue.

- [x] This is a checked box. **Do not leave spaces around the `x`!**

## Issue with the same problem
- [ ] I have searched for other issues with the same problem.

## Hosting
- [x ] Is it a local machine?

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [x ] Technical Help
- [ ] Other

### Operating System? (include version)
Linux 

## Running Mode
- [ ] instaboost.exe
- [x] example.py , mode (0)?
- [ ] instaboost.py
- [ ] Other

## Python Version?
2.4

## Description

Hi, 
Does this software support a direct message feature? If yes. how can i use it !?

**Comment by Morocco-germany** [comment]
it´s a possible to create this function in your Bot !? 
like this: 
https://github.com/LevPasha/Instagram-API-python/issues/181

**Comment by andrewsegas** [comment]
No, as you see this example they're using API, and here in our robot we don't use instagram API.
